server/config.js

```diff
@@ -51,9 +51,15 @@ const apis = {
   },
 }
 
+const useOfForce = {
+  ui_url: process.env.USE_OF_FORCE_URL || 'https://www.bbc.co.uk',
```

Comment:
the default port for use-of-force is 3000, but this will conflict with the default port for this app. 
So could leave the default blank here:
`ui_url: process.env.USE_OF_FORCE_URL`

To test it, you will need to provide a value in start-feature in package.json - to enable the link to display

Reply:
ok